Training/Easy/chuck_norris.rb

```diff
@@ -27,25 +27,24 @@ def split_series(array)
     result
```

Comment:
You can use `each_with_object` in the `split_series` method as well

Also, use meaningful variable names - `a` and `c` are not meaningful enough